integration/query_comparison_big_numbers_test.go

```diff
@@ -0,0 +1,77 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"golang.org/x/exp/constraints"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+
+	"github.com/FerretDB/FerretDB/integration/shareddata"
+)
+
+// Values stores shared data documents as {"_id": key, "value": value} documents.
+type values[idType constraints.Ordered] struct {
```

Comment:
Why did we copy all that?

I think it should be enough to add a few test cases to https://github.com/FerretDB/FerretDB/blob/main/integration/query_comparision_test.go